ros_installation.md

```diff
@@ -170,23 +170,43 @@ We do not recommend this approach for beginners!
 </div>
 
 To select a different release of Gazebo than the one officially supported by
-your ROS distribution you must **manually compile**
-[`ros_gz`](https://github.com/gazebosim/ros_gz) from source. This rule applies for
-every  ROS package that uses a Gazebo library. This approach may also require
-that you modify your  ROS or Gazebo source code to support this compilation.
+your ROS distribution you must either **use non ROS official Gazebo binary packages** or
+**manually compile**
+[`ros_gz`](https://github.com/gazebosim/ros_gz) from source (this rule applies for
+every ROS package that uses a Gazebo library):
+
+  * **Use non ROS official ros_gz binary packages:** *packages.osrfoundation.org* provides
+    binary packages for `ros_gz` for some combinations of Gazebo and ROS that are different
+    from the existing in ROS. In some circumstances these packages can be outdated or
```

Comment:
Is "existing" saying the officially recommended / supported combination? Using the same wording would be less confusing
"these" is saying the non-official ones, right? It is easily misinterpreted because it follows right after the last words in the previous sentence that's saying the official ones.
```suggestion
    from the officially supported combination. In some circumstances the non-official packages can be outdated or
```